tests/Fake/FakeParamsClass.php

```diff
@@ -5,9 +5,11 @@ class FakeParamsClass
 {
     public $array;
     public $empty = 'not null';
-    public function __construct(array $array, $empty)
+    public $items;
+    public function __construct(array $array, $empty, \stdClass ...$items)
```

Comment:
We may want to skip this class. 

Also a test condition need to be satisfied to make it work on 5.5 . Variadics is added only after 5.6 . If you can make a patch happy to have. Else I will dig into this.

Thank you once again for the PR. It looks good to me as all tests are also passing apart on 5.5 .